File: Python/truck.py
from landway import Landway

class Truck(Landway):

    def __init__(self, vehicle_id, vehicle_name, engine_type, capacity):
        super().__init__(vehicle_id, vehicle_name, engine_type)
        self.cargo_capacity = capacity

    def display_details(self):
        super().display_details()
        print("Cargo Capacity  :", self.cargo_capacity, "kg")

    def start_journey(self):
        print("Truck Journey Started Successfully...")

File: Java/Truck.java
class Truck extends Landway
{
    private double cargoWeight;

    Truck(int id, String name, String engineType, double cargoWeight)
    {
        super(id, name, engineType);
        this.cargoWeight = cargoWeight;
    }

    public void displayDetails()
    {
        super.displayDetails();
        System.out.println("Cargo Weight   : " + cargoWeight + " kg");
    }

    @Override
    void startJourney()
    {
        System.out.println("Truck Journey Started Successfully...");
    }
}